lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -219,6 +222,41 @@
                 },
 
                 /**
+                 * Render the edit team form.
+                 */
+                editTeam: function (topicID, teamID) {
+                    var self = this;
+                    this.getTeamsView(topicID).done(function (teamsView) {
+                        self.getTeam(teamID, false).done(function(team) {
+                            self.mainView = new ViewWithHeader({
+                                header: new HeaderView({
+                                    model: new TeamsHeaderModel({
+                                        description: gettext("You can edit the details of a team here, and should separately let your team members know about changes."),
```

Comment:
"If you edit any team details, you should let the team's members know about the changes."